raiden/api/python.py

```diff
@@ -815,3 +817,35 @@ def get_blockchain_events_channel(
             ))
         returned_events.sort(key=lambda evt: evt.get('block_number'), reverse=True)
         return returned_events
+
+    def create_monitoring_request(
```

Comment:
nitpick: I have already told you in person that I don't agree with the placement of this function here. I don't believe it makes sense (semantically) to reside in `python/api.py` since I see it as the connection between the rest api and the raiden service. And `create_monitoring_request` is not exposed to the user via REST.

I am just stating my disagreement. I don't think the PR should be blocked for it as it does the job it should do.